README.md

```diff
@@ -4,11 +4,73 @@ This repository contains the source code for SKDB and its associated
 components.
 
 
-## Dependencies
+## Running Prepackaged SKDB
 
-You will need at least the following installed:
+You can run the latest release of SKDB's development server with Docker:
 
- * LLVM 15.
- * wasm-ld 15 (this is part of LLVM, but your distribution might split it into
-   a separate package: your wasm-ld version must match your LLVM version).
- * typescript 5.2 or later.
+```sh
+docker run -it -p 3586:8080 skiplabs/skdb-dev-server
+```
+
+The three SKDB packages (`skdb`, `skdb-dev`, and `skdb-react`) are all
+available as normal NPM packages.
+
+
+## Building From Source
+
+It is recommended that you use the prepackaged version of SKBD in most
+situations.
+
+For those who need to build from source, you will need a number of
+dependencies installed: SKDB's
+[`Dockerfile`](https://github.com/SkipLabs/skdb/blob/main/Dockerfile) contains
+the definitive list for Ubuntu, and will give users of other operating systems
+a good idea of what is needed. Note that you will need LLVM 15, and that other
+tools relevant to LLVM need to be compatible with LLVM 15.
+
+If your distribution/OS doesn't include typescript, you can install your own
+with `npm install --prefix=/path/to/install typescript`.
+
+
+### Build steps
+
+These build steps can be simplified for your first build, but for simplicities
+sake we assume that you are doing multiple builds, and need to clean artefacts
+from previous builds: this is also safe, if unnecessary, for your first build.
+
+First build and install the bootstrap compiler:
+
+```sh
+cd /path/to/skdbrepo/compiler && make clean && prefix=/path/to/install make install STAGE=0
+```
+
+Then build the rest of the system ensuring that `/path/to/install` from the
+previous step is in your `$PATH`:
```

Comment:
If the advice is to use a custom install prefix, it is just as easy to tell people to ensure that `/path/to/skdbrepo/compiler/stage0/bin` is in `PATH`, and change `make install STAGE=0` to `make default STAGE=0`.

Reply:
Excellent idea! Fixed in https://github.com/SkipLabs/skdb/pull/80/commits/fb935fea05d0be47bf41d0b0462b1547e00972c8.

Reply:
In case it is helpful, my current workflow involves running the following command:
```
(set -a; SKARGO_PROFILE=dev; STAGE=C; PATH=$PWD/compiler/stage$STAGE/bin:$PATH; set +a; make clean && cd compiler && make clean test && cd .. && make test)
```

Reply:
What does `STAGE=C` do?

Reply:
Stage{A,B,C} are like stage{0,1,2} but where stage0 builds the bitcode checked in to the compiler/bootstrap dir, stageA uses the skc on the host found in PATH. This avoids things going out of sync when the runtime has changed since the bootstrap bitcode has been updated. See https://github.com/SkipLabs/skdb/pull/65 for more. It is useful for testing changes to the compiler and runtime system, but probably irrelevant for people just wanting to build from source.

Reply:
Right, got it!